schema/data/dev/005_cif_reporting_requirement.sql

```diff
@@ -23,16 +26,16 @@ do $$
       update cif.form_change set new_form_data =
       json_build_object(
         'projectId', temp_row.form_data_record_id,
-        'reportDueDate', now(),
-        'submittedDate', now(),
+        'reportDueDate', end_of_day_today,
+        'submittedDate', end_of_day_today,
```

Comment:
In real data, the report due date has an end-of-day timestamp. If we use `now()` here, then any reports due today have `Late` status

Reply:
Using `end_of_day_today` was causing some end-to-end tests to fail. If we continue to use it, the value will always change and we won't be able to test it properly. Additionally, it was making happo unhappy :)